__tests__/app.test.js

```diff
@@ -61,6 +61,53 @@ describe("GET /api", () => {
             })
             expect(Object.keys(response.body.endpoints).length).toBe(Object.keys(endpointsActual).length)
         });
-      })
-  });
+    });
+});
+test('404: reponds with Not found when given a non existent endpoint', () => {
+    return request(app)
+    .get('/api/nonExistentEndpoint')
+    .expect(404)
+    .then((response) => {
+        expect(response.body.msg).toBe('404: Not found')
+    });
+});
+})
+
+
+describe('GET /api/articles/:article_id', () => {
+    test('200: responds with an article object corresponding to the correct ID', () => {
+        const expected = {
```

Comment:
Good stuff on asserting for the specific requested article being returned. 👍  However, if the shape of article changes (eg another property is added) in the future then this test could fail. I would consider using a different way of asserting the correct article is returned but allows the flexibility of changes.